packages/sites/src/use-sites-list-sorting.tsx

```diff
@@ -242,6 +242,10 @@ function sortAlphabetically< T extends SiteDetailsForSorting >(
 	b: T,
 	sortOrder: SitesSortOrder
 ) {
+	if ( ! a?.title || ! b?.title ) {
```

Comment:
Should we mock this scenario in a test?

Reply:
I also checked the case when the title is an empty string `''` and it showed the same order as production 👍 .